examples/StandardFirmataBLE/bleConfig.h

```diff
@@ -5,20 +5,37 @@
  * need a unique ble local name (see below). If you are using another supported BLE board or shield,
  * follow the instructions for the specific board or shield below.
  *
- * Make sure you have the Intel Curie Boards package v1.0.6 or higher installed via the Arduino
+ * Make sure you have the Intel Curie Boards package v2.0.2 or higher installed via the Arduino
  * Boards Manager.
  *
  * Supported boards and shields:
  * - Arduino 101 (recommended)
  * - RedBearLab BLE Shield (v2)  ** to be verified **
  * - RedBearLab BLE Nano ** works with modifications **
+ * - Adafruit Feather M0 Bluefruit LE
  *
  *================================================================================================*/
 
 // change this to a unique name per board if running StandardFirmataBLE on multiple boards
 // within the same physical space
 #define FIRMATA_BLE_LOCAL_NAME "FIRMATA"
 
+/*
+ * Arduino 101
+ *
+ * Make sure you have the Intel Curie Boards package v2.0.2 or higher installed via the Arduino
+ * Boards Manager.
+ *
+ * Test script: https://gist.github.com/soundanalogous/927360b797574ed50e27
+ */
+#ifdef _VARIANT_ARDUINO_101_X_
+// After conversion to units of 1.25ms, both values must be between
```

Comment:
We can remove all the comments here since the user doesn't need to be concerned with the conversion anymore. I'd just update the comments on lines 34 and 35 to say `// 8ms` and `// 30ms` like you did for the Bluefruit configuration.

Reply:
I'd say it's still useful to know that, under the hood, the values are converted to multiples of 1.25ms -- if only to explain why using the value 8 actually gets you an interval of 7.5ms.

Reply:
ok sgtm